Documentation/teaching/labs/interrupts.rst

```diff
@@ -703,13 +703,22 @@ device driver.
 1. Request the I/O ports
 ------------------------
 
+To start with, we aim to allocate memory in the I/O space for hardware
+devices. We will see that we cannot allocate space for the keyboard
+because the designated region is already allocated. Then we will allocate
+I/O space for unused ports.
+
 The *kbd.c* file contains a skeleton for the keyboard driver. Browse
 the source code and inspect :c:func:`kbd_init`. Notice that the I/O
 ports we need are I8042_STATUS_REG and I8042_DATA_REG.
 
-Request the I/O ports int :c:func:`kbd_init` and make sure to check
-for errors and to properly clean-up in case of errors. Also add code
-to release the I/O ports in :c:func:`kbd_exit`.
+Follow the sections maked with **TODO 1** in the skeleton. Request the I/O
+ports in :c:func:`kbd_init` and make sure to check for errors and to properly
+clean-up in case of errors. Also, add code to release the I/O ports in
+:c:func:`kbd_exit`.
+
+.. note:: You can review the `Request access to I/O ports`_ section before
+proceeding.
```

Comment:
This should be indented, otherwise it will appear on a new line outside of the note box.
Check https://linux-kernel-labs.github.io/refs/pull/166/merge/labs/interrupts.html#request-the-i-o-ports

Reply:
Fixed.